src/components/core/core.js

```diff
@@ -247,7 +239,8 @@ export default class Core extends UIObject {
     containers.map(this.appendContainer.bind(this))
     this.trigger(Events.CORE_CONTAINERS_CREATED)
     this.renderContainers()
-    this.setupMediaControl(this.getCurrentContainer())
+    this.activeContainer = containers[0]
```

Comment:
Is it safe to assume that `0` will always the active container? where does this policy is enforced?

Reply:
@leandromoreira this is a convention already in place. If you look at the previous `getCurrentContainer` implementation, we always assumed that the current container was the first container unless otherwise stated.

Since the containers are only created when there's at least a single source, it's safe to assume that the first container should be initially the active one. If there is more than one source, it's still valid to assume that, since the media control should refer to a single container to control it. Also, when there's more than one source, we should assume that a plugin would take control over what container is currently active, for example, a picture-in-picture plugin or a grid plugin.